### PR TITLE
E402 - module level import not at top of file

### PR DESCRIPTION
**Pull request by guedou**
This PR of the #1277 serie add fixes for E402. When the changes were obvious, I did not use noqa. Otherwise, I believe that it is safer to comment  the offending lines.

### CODE REVIEW

File: scapy/layers/inet6.py [hunk 1]
```diff
@@ -43,33 +43,33 @@
...
 
-from scapy.arch import get_if_hwaddr
-from scapy.as_resolvers import AS_resolver_riswhois
-from scapy.base_classes import Gen
-from scapy.compat import chb, orb, raw, plain_str
-from scapy.config import conf
-import scapy.consts
+from scapy.arch import get_if_hwaddr  # noqa: E402
+from scapy.as_resolvers import AS_resolver_riswhoi  # noqa: E402s
+from scapy.base_classes import Ge  # noqa: E402n
```
Comment (gpotter2) [inline scapy/layers/inet6.py]: Few glitches over here

Comment (guedou) [reply scapy/layers/inet6.py]: Thanks!
I managed to fix it before boarding =)

**Comment by p-l-** [comment]
I don't understand why we cannot move **all** module level imports at top of file? If we cannot, I'd rather keep E402 as a global exception than have those `# noqa:` everywhere. What do you think?

**Comment by guedou** [comment]
I am afraid that fixing everything in bulk might blindly break things. Some are used to auto detect things.

I am ok with the current state of the PR, as it will enforce better usage in future code.

**Comment by gpotter2** [comment]
I’d say as @p-l- : I’m pretty sure most of those could be quite easily fixed.

Some seems necessary (see the comment above the change in tls/crypto/ciphers) but a lot are simply misplaced (e.g six imports)

**Comment by guedou** [comment]
Could you help me identify the imports that can be moved? It did not manage to do more than what I already did. Thanks.

**Comment by guedou** [comment]
@gpotter2 can you have a look at the modifications that I made to `scapy/arch/windows/__init__.py`. I don't think that we can remove the two noqa.

**Comment by gpotter2** [comment]
I agree, it seems this one is mandatory

**Comment by guedou** [comment]
@gpotter2 @p-l- This PR is ready to be reviewed. I think that I removed all the unnecessary `noqa`.

**Comment by p-l-** [comment]
@guedou sorry, can you rebase, I cannot merge it. Great work, once again!

**Comment by guedou** [comment]
Conflict fixed.
Self merging this PR as it was approved by @gpotter2 and @p-l-

**Pull request merged**